.eslintrc.js

```diff
@@ -42,9 +42,8 @@ module.exports = {
 
         // bind or arrow function in props causes performance issues
         // (but we currently use them in some places)
-        "react/jsx-no-bind": ["warn", {
-            "ignoreRefs": true,
-        }],
+        // It's disabled here, but we should using it sparingly.
+        "react/jsx-no-bind": "off",
```

Comment:
I've disabled this expecting that reviewers will complain loudly if the usage seems wrong, and to make the linter actually happy with the changeset in this PR.

Reply:
We use it in several other places as well. I didn't seem to mention it, but that comment applies to the future as well.